website/mkdocs.yml

```diff
@@ -45,4 +45,13 @@ nav:
   - Report Bugs & Feedbacks: feedbacks.md
   - Contributor Guides:
     - Developer: developer.md
+    - How to:
+       - how to build and debug with VS2019: "how to build and debug with VS2019.md"
+       - how to compile with cmake: "how to compile with cmake.md"
+       - how to connect with anki: "how to connect with anki.md"
+       - how to customize the opencc: "how to customize the opencc.md"
+       - how to find out the latest qt version and module in github qt action: "how to find out the latest qt version and module in github qt action.md"
+       - how to use .clang-format to format the code: "how to use .clang-format to format the code.md"
+       - how to use breadpad crash analysis: "how to use breadpad crash analysis.md"
+       - How to use ocr together with GoldenDict: "How to use ocr together with GoldenDict.md"
```

Comment:
"how to connect with anki" can be merged into "special usages -> anki integration"

"how to ocr" -> "special usages -> OCR integration"

seems better

Reply:
done